rundeckapp/src/main/groovy/org/rundeck/app/authorization/BaseAuthContextProvider.groovy

```diff
@@ -91,4 +92,21 @@ class BaseAuthContextProvider implements AuthContextProvider {
             authorizationService.getAuthorizationForSubject(new SubjectUserAndRoles(subject))
         )
     }
+
+    @Override
+    UserAndRolesAuthContext getAuthContextForUrnProject(String project) {
+        if (!project) {
+            throw new RuntimeException(
+                    "getAuthContextForUrnProject: Cannot get AuthContext without project: " +
+                            "${project}"
+            )
+        }
+
+        Subject subject = AclsUtil.getSubjectUrnForProject(project)
+
+        return new SubjectAuthContext(
+                subject,
+                authorizationService.getAuthorizationForSubject(new SubjectUserAndRoles(subject))
```

Comment:
this should probably use getProjectAuthorizationForSubject so that Project level ACLs can be loaded

Reply:
@ltamaster

Reply:
yes I changed that in the last commit

Reply:
oh sorry, for some reason the git diff I was viewing was not up to date